docs/platform/howto/list-authentication.md

```diff
@@ -2,9 +2,10 @@
 title: Authentication methods
 ---
 
-Browse through instructions for common Aiven platform tasks related to
-managing your password or other authentication methods.
+Users can authenticate to the Aiven platform using a password, single sign-on (SSO), or [tokens](/docs/platform/concepts/authentication-tokens). The available authentication methods depend on the organization's [authentication policy](/docs/platform/howto/set-authentication-policies).
```

Comment:
What about 3rd Party Authentication such as GitHub?

Reply:
This isn't meant to be an exhaustive list of all the options - third party providers are still SSO.

Reply:
They are also listed [here](https://aiven.io/docs/platform/howto/add-authentication-method) and [here](https://aiven.io/docs/platform/howto/set-authentication-policies).